bcda/api/alr_test.go

```diff
@@ -57,7 +57,7 @@ func (s *AlrTestSuite) TestAlrRequest() {
 	ad := auth.AuthData{ACOID: s.acoID.String(), CMSID: *aco.CMSID, TokenID: uuid.NewRandom().String()}
 
 	ctx := context.WithValue(req.Context(), auth.AuthDataContextKey, ad)
-	ctx = middleware.NewRequestParametersContext(ctx, middleware.RequestParameters{ResourceTypes: []string{"Patient", "Observation"}})
+	ctx = middleware.NewCtxRequestParams(ctx, middleware.RequestParameters{ResourceTypes: []string{"Patient", "Observation"}})
```

Comment:
Following the `Get` name convention Id like to see this as `Set`, ie `SetCtxRequestParams` (or `SetRequestParamsInCtx`).

Reply:
I switched these to `set`, although it seems that this is a heavily debated topic in the go community. Some folks say that the usage of getters and setters aren't really necessary unless we are using along an interface, etc. For the sake of readability, we can just keep them as is for now.